source/process/definitions.rst

```diff
@@ -0,0 +1,64 @@
+.. _definitions:
+
+Levels of Feature Quality, Development, and Support
+===================================================
+
+This document defines the terminology that is used when communicating status of features to the Mattermost community. The terminology applies to all editions of Mattermost platform and to Mattermost apps.
+
+.. contents::
+  :local:
+  :backlinks: top
+
+Feature Quality Levels
```

Comment:
For each category of levels propose adding an intro sentence explaining what each means.